packages/contact-center/store/src/store.ts

```diff
@@ -11,20 +19,31 @@ class Store {
     makeAutoObservable(this, {cc: observable.ref});
   }
 
-  init(webexConfig: any, access_token: string): Promise<void> { 
+  registerCC(webex: WithWebex['webex']): Promise<Profile> {
+    this.cc = webex.cc;
+    console.trace(this.cc.register);
+    return this.cc.register().then((response: Profile) => {
+      this.teams = response.teams;
+      this.loginOptions = response.loginVoiceOptions;
+    }).catch((error) => {
+      console.error('Error registering contact center', error);
```

Comment:
Won't we need to reject the promise in error case? Also will return take of resolving the promise.